### ISSUE TITLE
LABEL LEAKAGE During Training

### ISSUE THREAD

**Issue by MokukoHanehi** [rmokady/CLIP_prefix_caption#15]
In code: https://github.com/rmokady/CLIP_prefix_caption/blob/main/parse_coco.py#L26, you refer to the validation set to get the training samples. Also, in your file: https://drive.google.com/file/d/1D3EzUK1d1lNhD2hAvRiKPThidiVbP2K_/view?usp=sharing, there are validation captions (labels) in the training dataset, which results in obvious label leakage since the model will be trained on the validation samples. Actually if we train the model for more than 10 epochs (e.g. 100 epochs), the results in the validation set can reach an unreasonably high scores. Can you explain this?

**Comment by rmokady** [comment]
Hi @MokukoHanehi,
We didn't provide the validation set to make things simple
So I'm not sure which validation set you refer to

Is it possible the use of Karpathy et el. split confuses you? that we don't use the exact train and validation as the official COCO?

**Comment by MokukoHanehi** [comment]
Thank you for your reply. I am sorry that I misunderstood the train-val split of your paper. I use the official val set of COCO to evaluate your model, which contains some samples in the train split of Karpathy et al., therefore the result was unreasonably high. Thank you again for your kindly reply.
